torch/lib/c10d/ProcessGroup.hpp

```diff
@@ -137,6 +135,10 @@ class ProcessGroup {
 
     OpType retrieveOpType();
 
+    // Keeps track of the future responsible for profiling owner creation
```

Comment:
what does "profiling owner creation" mean? Do you just mean that this is a future that is complete when the profiling has finished?